CRD-DTR/HomeOxygenTherapy/R4/resources/Questionnaire-R4-HomeOxygenTherapyFaceToFace.json

```diff
@@ -718,15 +767,42 @@
         {
           "linkId": "8.3",
           "type": "text",
-          "text": "Explain why any of the alternative treatments were not considered?",
+          "text": "If No: Were any of the above treatments considered?",
```

Comment:
This question should probably be of type boolean, but if it's kept as text, then the enableWhen in `8.3.1` needs to change to `answerText`